apps/src/music/appConfig.ts

```diff
@@ -1,12 +1,14 @@
 import {queryParams} from '@cdo/apps/code-studio/utils';
-import {BlockMode} from './constants';
+import {BlockMode, baseAssetUrl} from './constants';
+
+type ConfigValues = {[key: string]: string};
 
 // Static config values passed in via a call to setAppConfig.
 // If these are set, we ignore URL parameters.
-let configValues = null;
+let configValues: ConfigValues | null = null;
 
 // Called to set config values.
-export function setAppConfig(values) {
+export function setAppConfig(values: ConfigValues) {
```

Comment:
Not required for this change, but could we remove this actually? I don't think we still use this right?

Reply:
Oh good call.  Let's do in a follow-up.

Reply:
Prepped in https://github.com/code-dot-org/code-dot-org/pull/57100.